mock/py/mockbuild/plugins/bind_mount.py

```diff
@@ -43,8 +44,9 @@ def __init__(self, plugins, conf, buildroot):
 
     @traceLog()
     def _bindMountCreateDirs(self):
-        create_dirs = self.config['plugin_conf']['bind_mount_opts']['create_dirs']
         for srcdir, destdir in self.bind_opts['dirs']:
-            if create_dirs:
+            if os.path.isdir(srcdir):
                 mockbuild.util.mkdirIfAbsent(srcdir)
-            mockbuild.util.mkdirIfAbsent(self.buildroot.make_chroot_path(destdir))
+                mockbuild.util.mkdirIfAbsent(self.buildroot.make_chroot_path(destdir))
+            else:
+                mockbuild.util.do(['/usr/bin/touch', self.buildroot.make_chroot_path(destdir)])
```

Comment:
can be replaced by `mockbuild.util.touch(self.buildroot.make_chroot_path(destdir))`